server/index.js

```diff
@@ -33,7 +33,12 @@ export default class Server {
     this.http = null
     this.config = getConfig(this.dir, conf)
     this.dist = this.config.distDir
-    this.buildStats = !dev ? require(join(this.dir, this.dist, 'build-stats.json')) : null
+    try {
+      this.buildStats = !dev ? require(join(this.dir, this.dist, 'build-stats.json')) : null
+    } catch (e) {
+      console.error('No build-stats.json found. Did you forget to run `next build`?')
```

Comment:
I'd like to do this in our `bin/*` files.
For an example, do this check here: https://github.com/zeit/next.js/blob/v3-beta/bin/next-start#L55